### PR TITLE
302 - add Ids progress chart themes (need to merge 276 first)

### LINKED ISSUE #302: ProgressChart: Add missing theme styles
**Describe the bug**
We noticed that the themes for IdsProgressChart are not totally implemented.

**To Reproduce**
1. Go to http://localhost:4300/ids-progress-chart/
2. Click on '....' and change between the 6 themes
3. Compare colors to the same themes on https://main-enterprise.demo.design.infor.com/components/completion-chart/example-colors.html

**Expected behavior**
Colors match

### PR DESCRIPTION
**Pull request by vivz753**
**Explain the details for making this change. What existing problem does the pull request solve?**

- added colors for respective themes, class/new:light/dark/contrast
- refactored slots to ids-icon element which can be set by icon attribute
- adjust icon size based on chart size

**Related github/jira issue (required)**:

- Closes #302 

**Steps necessary to review your pull request (required)**:

- `npm run start:watch`
- http://localhost:4300/ids-progress-chart
- click through the themes
- can also set/change/remove icons through DOM functions

**Included in this Pull Request**:
- [ ] An e2e or functional test for the bug or feature.
- [ ] A note to the change log.

### CODE REVIEW

**Comment by tmcconechy** [comment]
#276 is merged do you need to do anything?

File: test/ids-icon/ids-icon-func-test.js [hunk 1]
```diff
@@ -97,7 +97,7 @@ describe('IdsIcon Component', () => {
     document.body.appendChild(container);
     icon.language = 'ar';
     expect(icon.isFlipped('previous-page')).toBeTruthy();
-    expect(icon.template()).toContain('class="flipped"'); // Class list doesnt work on SVG?
+    expect(icon.template()).toContain('class="flipped"');
```
Comment (vivz753) [inline test/ids-icon/ids-icon-func-test.js:100]: Works now after the changes made in `ids-icon.js` line 78

File: src/ids-progress-chart/ids-progress-chart.d.ts [hunk 2]
```diff
@@ -14,7 +14,7 @@ export default class IdsNotificationBanner extends IdsElement {
   progressLabel: string;
 
   /** Sets the size of the chart */
-  size: 'small' | 'large';
+  size: 'small' | 'normal';
```
Comment (vivz753) [inline src/ids-progress-chart/ids-progress-chart.d.ts:17]: switched to use the appropriate `ids-icon` size

**Comment by vivz753** [comment]
Just needed to merge main. You beat me to it. Just pushed a small change to the `doc/CHECKLIST.md` and ready for approvals now after checks finish

**Pull request merged**